packages/instantsearch.js/src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -316,6 +316,48 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
     ).toHaveBeenCalledWith(expect.objectContaining({ page: 3 }));
   });
 
+  it('goes to previous page correctly when state is mismatched', () => {
```

Comment:
A mix of formats is being used in the `it` test names so went with this as a starting point. Probably can be better than this tho, so open to suggestions.